runtime/src/main/java/org/corfudb/runtime/object/VersionLockedObject.java

```diff
@@ -128,7 +121,7 @@ public void optimisticCommitUnsafe(long version) {
         this.version = version;
         // TODO: fix the stream view pointer seek, for now
         // read will read the tx commit entry.
-        sv.next();
+        //sv.seek(version);
```

Comment:
The question is: post commit, do we need to seek to the right position at the stream ? it is commented out.